packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -153,7 +155,9 @@ class CloudFormation {
   }
 
   readStackEvents(stackName) {
-    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME);
+    ++this.readStackEventsCalls;
+    const waitTimeInMs = Math.min((2 ^ this.readStackEventsCalls) * CFN_POLL_TIME, CFN_POLL_TIME_MAX);
+    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), waitTimeInMs);
```

Comment:
nit: we can increment this.readStackEventsCalls afterwards so that it starts with  CFN_POLL_TIME